### ISSUE TITLE
Focus state of 'more' link doesn't follow Design System guidance

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#222]
The focus state of the "Find out more" link on the [homepage](https://govuk-brand-guidelines.netlify.app/) is not very visible.

It changes from `#1E71B8` on `#FFFFFF` to `#0B0C0C` on `#FFDD04`. That's a contrast ratio change of 3.8:1 and 1.3:1. The blue to black change is just about good enough to pass WCAG requirements (as it's more than 3:1). But this doesn't follow our usual focus state guidance to have a thicker bottom border. That thicker border would improve accessibility of the focus state. Although probably not if it extends to the bottom because that would be black on blue. So maybe better extend further up?

(This issue is also on the Design System website with the "Get started" link on the homepage.)

## Who needs to be involved

Developer, designer

## Who needs to review

Anyone in the squad
